corehq/apps/app_manager/views/download.py

```diff
@@ -374,6 +374,7 @@ def download_index_files(app):
                  for path in app._attachments
                  if path.startswith('files/')]
     else:
+        app.set_media_versions(None)
```

Comment:
This technique is used [elsewhere](https://github.com/dimagi/commcare-hq/blob/2248aa3f03c7d0b808642e1242b7e57248d42a9f/corehq/apps/app_manager/views/download.py#L70).